smoketest/configs/basic-vyos

```diff
@@ -87,6 +87,14 @@ service {
                     start 192.168.0.20
                     stop 192.168.0.240
                 }
+                static-mapping TEST1 {
+                    ip-address 192.168.0.21
+                    mac-address 00:01:02:03:04:05
+                }
+                static-mapping TEST2 {
```

Comment:
This is a testcase for duplicate MACs with different IPs, can we also have a case for duplicate IPs with different MACs?

Reply:
Added in  0270932